File: Tugas_Praktikum_Pemograman_1_Muhammad_Aufa_Fitrianda_2310817210013/Modul_4/Bahasa_C/Prak404_2310817210013_MuhammadAufaFitrianda.c
#include <stdio.h>

int main() {
    int pilihan;
    float nilaiPertama, nilaiKedua;

    while (1) {
        printf("Pilih program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan Pilihan: ");
        scanf("%d", &pilihan);

        if (pilihan == 5) {
            printf("Terimakasih, telah menggunakan kalkulator Muhammad Aufa Fitrianda\n");
            break; // Keluar dari program jika pilihan adalah 5 (Exit)
        }

        if (pilihan < 1 || pilihan > 4) {
            printf("Input anda salah, silahkan coba lagi\n");
            continue;
        }

        printf("Masukkan nilai pertama: ");
        scanf("%f", &nilaiPertama);
        printf("Masukkan nilai kedua: ");
        scanf("%f", &nilaiKedua);

        switch (pilihan) {
            case 1:
                printf("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f\n", nilaiPertama, nilaiKedua, nilaiPertama + nilaiKedua);
                break;
            case 2:
                printf("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n", nilaiPertama, nilaiKedua, nilaiPertama - nilaiKedua);
                break;
            case 3:
                printf("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n", nilaiPertama, nilaiKedua, nilaiPertama * nilaiKedua);
                break;
            case 4:
                if (nilaiKedua != 0) {
                    printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n", nilaiPertama, nilaiKedua, nilaiPertama / nilaiKedua);
                } else {
                    printf("Pembagian oleh nol tidak diizinkan.\n");
                }
                break;
        }
    }

    return 0;
}

File: Tugas_Praktikum_Pemograman_1_Muhammad_Aufa_Fitrianda_2310817210013/Modul_4/Bahasa_Python/Prak404_2310817210013_MuhammadAufaFitrianda.py
while True:
    print("Pilih program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")
    pilihan = int(input("Masukkan Pilihan: "))

    if pilihan == 5:
        print(f"Terimakasih, telah menggunakan kalkulator Muhammad Aufa Fitrianda")
        break  

    if pilihan < 1 or pilihan > 4:
        print("Input anda salah, silahkan coba lagi")
        continue

    nilaiPertama = float(input("Masukkan nilai pertama: "))
    nilaiKedua = float(input("Masukkan nilai kedua: "))

    if pilihan == 1:
        hasil = nilaiPertama + nilaiKedua
        operasi = "Penjumlahan"
    elif pilihan == 2:
        hasil = nilaiPertama - nilaiKedua
        operasi = "Pengurangan"
    elif pilihan == 3:
        hasil = nilaiPertama * nilaiKedua
        operasi = "Perkalian"
    elif pilihan == 4:
        if nilaiKedua != 0:
            hasil = nilaiPertama / nilaiKedua
            operasi = "Pembagian"
        else:
            print("Pembagian oleh nol tidak diizinkan.")
            continue

    print(f"Hasil {operasi} antara {nilaiPertama:.2f} dengan {nilaiKedua:.2f} adalah {hasil:.2f}")
